peekingduck/pipeline/nodes/dabble/tracking.py

```diff
@@ -12,53 +12,66 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""
-Performs multiple object tracking for detected bboxes
-"""
+"""Performs multiple object tracking for detected bboxes."""
 
-from typing import Dict, Any
+from typing import Any, Dict
+
+from peekingduck.pipeline.nodes.dabble.trackingv1.detection_tracker import (
+    DetectionTracker,
+)
 from peekingduck.pipeline.nodes.node import AbstractNode
-from peekingduck.pipeline.nodes.dabble.utils.load_tracker import TrackerLoader
 
 
 class Node(AbstractNode):
-    """Node that uses bounding boxes detected by an object detector model
-    to track multiple objects.
+    """Uses bounding boxes detected by an object detector model to track
+    multiple objects.
+
+    Currently, two types of tracking algorithms can be selected: MOSSE, IOU.
 
-    There are types of trackers that can be selected: MOSSE, IOU.
-    Please view each tracker's script, or the "Multi Object
-    Tracking" use case documentation for more details.
+    Please view each tracker's script, or the "Multi Object Tracking" use case
+    documentation for more details.
```

Comment:
> The intention is to ask users to look at the python files for more details?

Not too sure about this one actually, I think because RTD omits non-Node documentations by default so the elaboration of the 2 trackers can only be found in their respective docstrings.

I've added a "under construction" object tracking benchmark page and changed the docstring to link to that instead.